src/background/actions/assessment-action-creator.ts

```diff
@@ -151,6 +151,10 @@ export class AssessmentActionCreator {
             Messages.Visualizations.DetailsView.Initialize,
             this.onDetailsViewInitialized,
         );
+        this.interpreter.registerTypeToPayloadCallback(
```

Comment:
Why is `this.onDetailsViewInitialized` triggered by two different message types?

Reply:
Thanks for pointing this out! Discussed this with Wahaj and decided to remove the initialize call from assessment as it isn't really needed.